source/c.txt

```diff
@@ -75,6 +76,37 @@ Connect to MongoDB Atlas
 See `Advanced Connections <http://mongoc.org/libmongoc/current/advanced-connections.html>`__
 for more ways to connect.
 
+Versioned API
+-------------
+
+You can use the `Versioned API
+<http://docs.mongodb.com/v5.0/reference/versioned-api/>`__ feature
+starting with MongoDB server version 5.0 and C Driver version 1.18. When
```

Comment:
I think the proper noun for server is suitable here since you're using the proper noun version of driver. There's a minor distinction between the two forms, but let's stick to the proper noun versions of both here.

Reply:
suggestion accepted